distribution/build.gradle

```diff
@@ -422,6 +422,7 @@ testClusters {
     if (System.getProperty('run.distribution', 'default') == 'default') {
       String licenseType = System.getProperty("run.license_type", "basic")
       if (licenseType == 'trial') {
+        setting 'xpack.eql.enabled', 'true'
```

Comment:
We enable it here only for trial, but we run integration tests under basic. That doesn't seem to be consistent. I would just put next to `pack.sql.enabled` until we figure out what to do with it.

Reply:
Updated to enable consistently with sql plugin for run task, seems logical.